README.md

```diff
@@ -0,0 +1,128 @@
+<div align="center"><h1>Welcome to the Bookworm's Haven</h1></div>
+
+#### Dive into the world of books with our online bookstore.
+
+Welcome to the Bookworm's Haven, a Java Spring Boot application that is designed to make your project development journey smoother and more efficient. This platform combines a rich set of features and modern technologies to create a secure and user-friendly e-commerce experience. Below, you'll discover the key elements and functionalities of the application.
+
+- [Introduction](#introduction)
+- [Technologies & Tools](#technologies)
+- [Functionalities](#functionalities)
+- [Getting Started](#setup)
+- [Video Guide](#video-instruction)
+- [Testing Tools](#usage-tools)
+- [Contribute](#contribute)
+
+<div id="introduction">
+    <br>
+    <p>
+        Welcome to the Bookworm's Haven, a delightful application created for book enthusiasts. It provides you with the ability to select the perfect book, search through our extensive library, and compile your order effortlessly. This project encompasses various domain models including Users, Roles, Books, Categories, Shopping Carts, Cart Items, Orders, and Order Items. Users can register, log in, explore books, add them to their shopping cart, and complete orders. Administrators have the authority to manage books, bookshelf sections, and oversee and modify order statuses.
+        This README will guide you through using the application and setting it up on your local machine.
+    </p>
+</div>
+<hr>
+
+<div id="technologies">
+    <h3> Technologies & Tools </h3>
+    <p>
+        Our web application is built on Spring Boot and utilizes a range of modern technologies and tools to ensure robustness and scalability, including:
+       <br> - Spring Boot Framework for the backend 
+       <br> - Spring Security for authentication and security
+       <br> - Spring Data JPA for database interaction
+       <br> - Swagger for API documentation
+       <br> - Maven for library management and project building
+       <br> - IntelliJ IDEA as the recommended development environment
+       <br> - Docker for running in an isolated environment on different platforms
+       <br> - Liquibase for managing database schema changes and revisions 
+       <br> - Spring Testing for quality assurance
+</p>
+</div>
+<hr>
+
+<div id="functionalities">
+    <h3> Functionalities </h3>
+    <p>
+        This project features two user roles, ROLE_USER (default) and ROLE_ADMIN, each with their respective capabilities:
```

Comment:
create some kind of mapping between functionalities and controller which is responsible for specific functionality